.travis.yml

```diff
@@ -22,8 +22,14 @@ script: ci/script.sh
 before_deploy: ci/before_deploy.sh
 
 deploy:
-  # deploy artifacts to Maven Central
   - provider: script
     script: ci/deploy.sh
     on:
       tags: true
+  - provider: releases
+    api_key: "${GH_TOKEN}"
+    file_glob: true
+    file: "./target/recheck-*.jar.*"
```

Comment:
No need to upload the JAR or anything else, we just only want a release with: `draft: true`.

Reply:
Just like this (we can probably also remove `skip_cleanup: true`): https://github.com/retest/recheck/commit/eaa1ec88538a7edcc708fabd202c1e6480be3e61